sdk/webpubsub/Microsoft.Azure.Functions.Worker.Extensions.WebPubSub/src/Models/Output/SendToConnectionAction.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// Operation to send message to a connection.
+    /// </summary>
+    public class SendToConnectionAction : WebPubSubAction
```

Comment:
I suppose we don't need any converters to use camel case because this is handled in the extension itself?
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Bindings/Output/RemoveConnectionFromGroupAction.cs#L12

Reply:
Suppose the common practice is not to mix use in-process and isolated-process extension packages. So either customers have to declare themselves or we do this for convenience.

See sample without built-in types and customer have to do in this way in early SignalR extension: https://github.com/Azure/azure-functions-dotnet-worker/blob/main/samples/Extensions/SignalR/SignalRFunction.cs